contracts/StablecoinConverter.sol

```diff
@@ -29,6 +30,9 @@ contract StablecoinConverter is EpochTokenLocker {
     /** @dev Maximum number of touched orders in auction (used in submitSolution) */
     uint constant public MAX_TOUCHED_ORDERS = 25;
 
+    /** @dev Fee charged for adding a token) */
+    uint constant public TOKEN_ADDITION_FEE_IN_OWL = 10*10**18;
```

Comment:
You could be using `10 ether` here which has the same meaning. Do you think that would be too confusing for the reader?

Reply:
+1 to using either.
Is there any value in being 100% correct and calcualating `TOKEN_ADDITION_FEE_IN_OWL` in the constructor with something like `10 * 10**owl.decimals()`? That is assuming OWL is ERC20 token with `decimals` property.

Reply:
I would prefer to use `10 ether` here. Feel free to commit this suggestion (it shouldn't affect anything else).  
```suggestion
    uint constant public TOKEN_ADDITION_FEE_IN_OWL = 10 ether;
```

Reply:
`10**owl.decimals()`

I decided for ether, instead of decimals, as I want it to be a constant not be changed by the constructor. Though I think that `ether` is also not soo good, as it implies a different nomination in eth instead of OWL

Reply:
ya... thats cool. Shall we leave it as it is?